src/pages/base/user.js

```diff
@@ -83,7 +84,7 @@ export default class BaseUserPage extends React.Component {
         </Header>
 
         <Page>
-          <Sidebar current_user={current_user} />
+          <Sidebar current_user={current_user} routing={routing} />
```

Comment:
Sidebar is an active component, connected to redux. It has access to `routing` already (just add it to selector)